src/routes/articles/delete-article.e2e.test.js

```diff
@@ -0,0 +1,86 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('DELETE /articles/:id', () => {
+
+	let user
+	let accessToken
+
+	before(async ()=>{
+
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+	
+	describe('Failure', () => {
+		
+		it('should return 400 if postId is not a number', async () => {
+			const expectedError = {
+				"error": {
+			  "message": "id must be a number"
+				},
+		   "status": "failed"
+		  }
+			return	fixtures.api()
+				.delete(`/api/v1/articles/sdsdsd`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400, expectedError)
+
+		})
+
+		it('should return 401 if request is not authenticated', async () => {
+			const post = await fixtures.insertPost({
```

Comment:
the request is not authenticated and won't work anyway so we don't have to insert a post

Reply:
yes please